### ISSUE TITLE
Unmarshal url query?

### ISSUE THREAD

**Issue by ivanjaros** [goccy/go-json#420]
Is it possible to un/marshal url query? Maybe with the field query functionality(I am not sure what they are for).

**Comment by ivanjaros** [comment]
There is the gorilla schema library but entire gorilla is archived now. Also it would be nice to have it all in one place. The query is essentially only map[string][]string so it should not be hard to parse.

**Comment by goccy** [comment]
I think that it is ok if encoding/json can parse it.
